beta/src/components/MDX/Sandpack/Console.tsx

```diff
@@ -106,8 +124,9 @@ export const SandpackConsole = () => {
                 <div
                   key={id}
                   className={cn(
-                    'last:border-none border-b dark:border-gray-700 text-md p-1 pl-2 leading-6 font-mono min-h-[32px]',
-                    `console-${getType(method)}`
+                    'last:border-none border-b dark:border-gray-700 text-md p-1 pl-2 leading-6 font-mono min-h-[32px] whitespace-pre-wrap',
+                    `console-${getType(method)}`,
+                    `${getColor(method)}`
```

Comment:
```suggestion
                    getColor(method)
```
This should also work?